### DISCUSSION TITLE
Getting IP through discord call

### DISCUSSION THREAD

**Discussion by Bohosul01** [hxr404/Discord-Console-hacks#40] [Q&A] [answered]
Just imaging. Does the staff mode open any way to get someone's IP address?

**Comment by hxr404** [comment, marked as answer]
No it doesn't. And I don't think it was ever possible to get someone's IP address using staff mode.
Discord has some WebRTC leaks which could allow them to get your real IP (even behind a VPN), but currently I'm not aware of any exploit which lets you (as an attacker) actually retrieve that IP.
Their old implementation was flawed and allowed literally anyone to grab your IP (not using staff mode, though), but now I think its fixed

**Comment by RobinRMC** [comment]
This thread should be moved to the Q&A category, and then marked as Answered.
